test/xref2/references_to_pages.t/run.t

```diff
@@ -1,9 +1,9 @@
 # References to pages and items in pages
 
   $ compile p.mld good_references.mli bad_references.mli
-  File "p.mld", line 6, characters 5-11:
+  File "p.mld", line 7, characters 5-11:
```

Comment:
This test is wrong, these references should be changed to: (the errors will disappear)
```
{!Good_references} {!Good_references.t}
```

Reply:
This test and the other tests failing in CI will be fixed in https://github.com/ocaml/odoc/pull/738